docs/Reference/Request.md

```diff
@@ -48,9 +52,6 @@ Request is a core Fastify object containing the following fields:
   schema and returns the serialized payload. If the optional
   `httpPart` is provided, the function will use the serializer
   function given for that HTTP Status Code. Defaults to `null`.
-- `context` - A Fastify internal object. You should not use it directly or
```

Comment:
we should keep this doc for now

Reply:
as we're going to keep it on `main/stable` shall we add some deprecation notice?

Reply:
I think it is not necessary on doc, there is the Nodejs warning already